src/app/dataset-view/additional-components/metadata-component/components/form-components/order-field/order-field.component.html

```diff
@@ -0,0 +1,36 @@
+<div [formGroup]="form">
+    <label class="form-label">{{ "Order" }} </label><br />
+    <div class="form-check">
+        <input
+            class="form-check-input"
+            type="radio"
+            formControlName="order"
+            [name]="'order'"
+            type="radio"
+            [value]="'none'"
+        />
+        <label class="form-check-label" for=""> None </label>
```

Comment:
A lot of repetition for 3 values. Try with some array and `ng-for`

Reply:
Refactored with *ngFor